libraries/standard.js

```diff
@@ -1,8 +1,33 @@
 const lib = new Map();
 
+// check if read values are null, undefined or blank (empty string)
+// dst: destination object, data: data item being processed, 
+// data_field: field being processed, skip_errors: boolean, errors: array of bad data items
+const null_check_function = (dst, data, data_field, skip_errors, errors) => {
+  // data or data_field are null or undefined, or data is empty object
+  if (!data || Object.keys(data).length === 0 || !data_field) {
+    console.log("No data or data field supplied")
+    return false
+  }
+  if (data[data_field]) {
```

Comment:
data_field wont be a string and instead will be the actual field, so you could do `if (data_field)` and `dst = data_field` instead. We'd be passing it like this for example `null_check_function(new_item.name, old_item, old_item.first_name, false, errors)`